components/profile/activityTab/HistoryRow.vue

```diff
@@ -9,7 +9,7 @@
             class="border border-k-shade w-[3.125rem] h-[3.125rem]"
             :alt="event.Item.name"
             :src="image"
-            :animation-src="animationUrl"
+            :animation-src="!image ? animationUrl : undefined"
```

Comment:
isn't it better to have one computed variable (DRY) ?

Reply:
i don't like to have computed refs for such simple things

i feel that it creates a mental overhead of linking the ref to where it is used in the template

matter of personal preference i guess

if you disagree in this case lmk and ill change


thanks :heart: